packages/editor/src/client/routes/pageEditor.tsx

```diff
@@ -228,7 +228,9 @@ export function PageEditor({id}: PageEditorProps) {
     }
 
     setChanged(false)
-    Alert.success(t('pageEditor.overview.pagePublished'), 2000)
+    publishData === new Date() || publishDate < new Date()
+      ? Alert.success(t('pageEditor.overview.pagePublished'), 2000)
+      : Alert.success(t('pageEditor.overview.pagePending'), 2000)
```

Comment:
You can change here `Alert` also to `Notification`. I think this got forgotten in the past.